src/workerd/io/compatibility-date.capnp

```diff
@@ -500,8 +500,7 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   globalFetchStrictlyPublic @51 :Bool
       $compatEnableFlag("global_fetch_strictly_public")
-      $compatDisableFlag("global_fetch_private_origin")
-      $experimental;
+      $compatDisableFlag("global_fetch_private_origin");
```

Comment:
Just want to confirm that we do not intend for this to become default on with some specific compat date?

Reply:
Currently - no
In the future - probably 😅